templates/remix-javascript/package.json

```diff
@@ -4,7 +4,7 @@
   "type": "module",
   "scripts": {
     "build": "remix build",
-    "dev": "remix dev",
+    "dev": "remix dev --manual",
```

Comment:
Can we default to manual if no command is provided now that the serve CLI that will be used does cache busting?

Reply:
We could, but then we'd need a way to opt out of `--manual` which we don't currently have.

For v2, Michael and I agreed that we probably want a lever like `--mode=watch`, `--mode=live-reload`, `--mode=hot` or something like that. naming TBD

Reply:
I just mean:
```js
if (!userProvidedCommand) {
  manual = true
}
```